archive/rpm.ksy

```diff
@@ -0,0 +1,472 @@
+meta:
+  id: rpm
+  file-extension:
+    - rpm
+    - srpm
+    - src.rpm
+    - drpm
+  xref:
+    pronom: fmt/795 # v3
+    wikidata: Q492650
+  license: CC0-1.0
+  ks-version: 0.9
+  endian: be
+doc: |
+  This parser isr for RPM version 3 which is the current version of RPM. There
```

Comment:
RPM is currently on _version 4_, not _version 3_.

Reply:
https://github.com/rpm-software-management/rpm/blob/master/doc/manual/format.md

```
This document describes the RPM file format version 3.0, which is used by RPM versions 2.1 and greater.
```

The major version in the header says 3 as well. Of course the version of the rpm tool is 4.x, but the file *format* is still version 3.

Reply:
I have clarified this in the description.